src/Hosting/Abstractions/src/Microsoft.AspNetCore.Hosting.Abstractions.csproj

```diff
@@ -11,28 +11,6 @@
     <IsTrimmable>true</IsTrimmable>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
Moved the internal metrics DI prototype from hosting abstractions to http abstractions. Hosting wasn't referenced by the rate limiting middleware project.

Reply:
Do we need to remove any references to Hosting.Abstractions in projects that were referencing it exclusively for the metrics-related APIs?

Reply:
No one was referencing Hosting.Abstractions for metrics. The reason I moved them to Http.Abstractions because everything references that.